lab-sarah/model/park.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const createError = require('http-errors');
+const debug = require('debug')('park:park-model');
+
+//have to have access to the dog in order to make park, because you can't have park without dogs
```

Comment:
This is technically not true -- you can have a park without dogs but can't have a dog without a park. However, once you have dogs, you want to populate the dogs array on the park with those dogs. This array can be empty if there are no dogs, however.